### ISSUE TITLE
Fetch failed loading: GET "http://localhost:3000/dashboard/blog?_rsc=1538b".

### ISSUE THREAD

**Issue by spacecat** [vercel/next.js#65387]
![Screenshot 2024-05-05 at 17 08 50](https://github.com/vercel/next.js/assets/866622/c90ac0ce-0710-40ec-bb64-452a822c3ff2)
![Screenshot 2024-05-05 at 17 09 23](https://github.com/vercel/next.js/assets/866622/f7dc7036-870b-403e-a24d-fea70a767d34)
### Link to the code that reproduces this issue

https://github.com/spacecat/nextjs-04

### To Reproduce

IMPORTANT: You need to enable "Log XMLHttpRequests" in Google Chrome DevTools Console settings:

<img width="552" alt="image" src="https://github.com/vercel/next.js/assets/866622/39caed81-fa87-477f-80d0-c901f9cf5553">

1. npm run dev
2. In Chrome, navigate to http://localhost:3000/dashboard
3. Click the "To Blog" link
4. Click the "To Dashboard" link

### Current vs. Expected behavior

I expected the DevTools console to NOT output any errors or messages that sound like errors.
Instead the DevTools console is outputting Fetch failed loading: GET messages.

### Provide environment information

```bash
Operating System:
  Platform: darwin
  Arch: arm64
  Version: Darwin Kernel Version 23.4.0: Fri Mar 15 00:12:41 PDT 2024; root:xnu-10063.101.17~1/RELEASE_ARM64_T8103
  Available memory (MB): 16384
  Available CPU cores: 8
Binaries:
  Node: 20.9.0
  npm: 10.1.0
  Yarn: 1.22.19
  pnpm: N/A
Relevant Packages:
  next: 14.2.3 // Latest available version is detected (14.2.3).
  eslint-config-next: 14.2.3
  react: 18.3.1
  react-dom: 18.3.1
  typescript: 5.4.5
Next.js Config:
  output: N/A
```

### Which area(s) are affected? (Select all that apply)

Developer Experience

### Which stage(s) are affected? (Select all that apply)

next dev (local)

### Additional context

I'm only getting these messages in Chrome on both my Mac and PC. Safari and Firefox don't give me these error messages.

**Comment by colemilne54** [comment]
I tested this on my Mac on Chrome and did not run into any errors. Here are the steps I followed

1. Cloned your repo
2. npm install
3. npm run dev
4. In Chrome, navigated to http://localhost:3000/dashboard
5. Clicked the "To Blog" link
6. Clicked the "To Dashboard" link

**Comment by spacecat** [comment]
@colemilne54 thanks for testing this out.

what if you contineously click on the links a lot of times, pause for some seconds, and then click on the links again a lot of times, then do you see the error?

I tested this again and it's random for me. I do not get the error everytime. And I'm not doing any full refresh in between - I just click the links. Here's a screenshot of my latest test, notice the timestamps:

<img width="1440" alt="image" src="https://github.com/vercel/next.js/assets/866622/9c43dba9-19c7-471b-bc9b-10cd90044ed4">

**Comment by spacecat** [comment]
Trying the same thing in Firefox - no XHR errors there 🤔:

<img width="1440" alt="image" src="https://github.com/vercel/next.js/assets/866622/9b39dd6a-14b7-4087-9057-cc10d7da7c02">

**Comment by spacecat** [comment]
Hi, am I the only one experiencing this? :)

**Comment by tmair** [comment]
I am also experiencing this issue on Chrome using MacOS. The application works fine.

I also noticed, that in the Network tab of the developer tools the request information is missing the Payload. Everything else is displayed correctly. 

So I think this might be an issue of the Chrome developer tools.

**Comment by spacecat** [comment]
Please triage this issue :)

**Comment by maxpowernz** [comment]
Getting the exact same thing too, is it the server action switching from front to backend request?

**Comment by maxpowernz** [comment]
> Hi, am I the only one experiencing this? :)

No, I see it to, response is ok, pretty sure its the usetransition hook, so not a next thing, more react

**Comment by marcoshernanz** [comment]
Thanks for the report. I reproduced the underlying `_rsc` cancellation, but the requests receive 200 responses, navigation completes, and no runtime error is emitted. The visible message comes from Chrome's optional XHR/fetch logger rather than Next.js error reporting, and production was clean. Since there is no user-visible failure to investigate, I'm closing this as non-actionable. If you encounter a navigation failure, please open a new issue with a current reproduction.
